core/src/main/java/io/paradoxical/cassieq/dataAccess/PointerRepositoryImpl.java

```diff
@@ -97,6 +97,15 @@ public RepairBucketPointer getRepairCurrentBucketPointer() {
         return pointer == null ? RepairBucketPointer.valueOf(0) : pointer;
     }
 
+    @Override
+    public void deleteAll() {
```

Comment:
yeah, def agreed. this is why i was mentiong we should refactor our repo layers cause its confusing right now